Resources/ServerInfo/_NF/Guidebook/Hiring.xml

```diff
@@ -0,0 +1,14 @@
+<Document>
+# Hiring Crew
+  <Box>
+    <GuideEntityEmbed Entity="ComputerStationRecords"/>
+    <GuideEntityEmbed Entity="ClothingHeadsetGrey" Caption="headset"/>
+    <GuideEntityEmbed Entity="IntercomCommon"/>
+  </Box>
+As the Captain of a Nanotrasen vessel, you have the authority to hire and fire crew members, and to delegate work at will. There are two ways to hire crew:
+- Use the [color=#a4885c]station records computer[/color] on your ship to open crew slots. Players that join later can spawn in on your ship.
+- Invite other players to join your crew through in-game communications.
+
+Crew membership is informally enforced - you don't need to sign any forms (but feel free to roleplay!), but it's a good idea to make sure your crew understand what you expect of them, and what they can expect from you (an hourly wage, a split of future rewards).  Remember, you're responsible for the well-being of your crew, and a captain's only as good as their word!
```

Comment:
> (an hourly wage, a split of future rewards)

imo this line makes it sound like you're required to split your profit with the crew, which is not the case: space law obligates you to pay only hourly wages any additional payments and bonuses are up you as a captain

Reply:
The goal is to say that they should be owed something, but that it's up to you to structure that.

I agree with you.  If I phrased it as a set of questions, would that help? How about this: "(An hourly wage? A split of future rewards? The promise of adventure?)"?

Also, as far as I can tell, Space Law only says that station crew must be paid a minimum wage.  Ancaps rejoice.

Reply:
> I agree with you. If I phrased it as a set of questions, would that help?

Yeah, that's more like it.

> Also, as far as I can tell, Space Law only says that station crew must be paid a minimum wage. Ancaps rejoice.

I believe there used to be a section in Mutiny that listed lack of hourly wages as a valid reason for crew to rebel.  Not any more afaict. It would seem I gotta catch up with current space law.